schematic/store/synapse.py

```diff
@@ -2082,22 +2297,28 @@ def upload_manifest_as_table(
             )
         # Load manifest to synapse as a CSV File
         manifest_synapse_file_id = self.upload_manifest_file(
-            manifest,
-            metadataManifestPath,
-            datasetId,
-            restrict,
+            manifest=manifest,
```

Comment:
I am seeing that you made similar changes in other parts of the code base by calling out the name of the parameter function. Is that a good practice?

Reply:
Yes. IMO Positional arguments are only good when there are 1, maybe 2 arguments you can pass into a function. Any more then it makes it significantly more difficult to skim the code and understand what is going on.

Reply:
Got it Bryan! Thanks for the explanation.